plugins/org.jkiss.dbeaver.model.jdbc/src/org/jkiss/dbeaver/model/impl/jdbc/data/JDBCCollection.java

```diff
@@ -455,6 +455,17 @@ private static JDBCCollection makeCollectionFromJavaArray(
         return new JDBCCollection(session.getProgressMonitor(), elementType, elementValueHandler, contents);
     }
 
+    private static boolean isBoxedObjectArray(Object array, Class<?> type) {
```

Comment:
Add `@NotNull/@Nullable` annotations, and **a** as variable name is unclear name